### PR TITLE
Add optional label to inputs

### PR DESCRIPTION
**Pull request by leigh-mil**
## Description
Uses the `optional` attribute in input macros to conditionally display the optional tag in form field labels. 
Let me know if any of the fields are incorrectly labeled optional or vice versa. I was going by what was required in our WTForms classes.

## Pivotal
https://www.pivotaltracker.com/story/show/168199417

## Screenshots
Screen Shot 2019-08-29 at 1 37 47 PM
Screen Shot 2019-08-29 at 1 37 51 PM
Screen Shot 2019-08-29 at 1 37 54 PM
Screen Shot 2019-08-29 at 1 40 00 PM
Screen Shot 2019-08-29 at 1 41 26 PM

### CODE REVIEW

File: styles/elements/_inputs.scss [hunk 1]
```diff
@@ -378,8 +378,7 @@ select {
     margin-left: $gap * 4;
 
     .usa-input {
-      input,
-      label {
+      input {
```
[the remark this answers is not in the document]
Comment (leigh-mil) [reply styles/elements/_inputs.scss]: yea, it was causing the optional tag to go to a second line in some cases

File: templates/components/multi_checkbox_input.html [hunk 2]
```diff
@@ -24,6 +24,10 @@
         <legend>
           <div class="usa-input__title">
             {{ field.label | striptags}}
+            {% if optional %}
+              &nbsp;
```
[the remark this answers is not in the document]
Comment (leigh-mil) [reply templates/components/multi_checkbox_input.html]: I could not get it to work with a regular space!

File: templates/components/text_input.html [hunk 3]
```diff
@@ -40,6 +40,10 @@
         <label for={{field.name}}>
           <div class="usa-input__title">
             {{ label }}
+            {% if optional %}
+              &nbsp;
```
Comment (dandds) [inline templates/components/text_input.html]: I think you missed one!

Comment (leigh-mil) [reply templates/components/text_input.html]: good catch!

**Pull request merged**